processing/src/main/java/org/apache/druid/segment/vector/FilteredVectorOffset.java

```diff
@@ -129,7 +129,8 @@ private void advanceWhileVectorIsEmptyAndPopulateOffsets()
         return;
       }
 
-      final ReadableVectorMatch match = filterMatcher.match(VectorMatch.allTrue(baseOffset.getCurrentVectorSize()));
+      final ReadableVectorMatch match = filterMatcher.match(VectorMatch.allTrue(baseOffset.getCurrentVectorSize()),
```

Comment:
Formatting is a little wonky

Reply:
this is the main reason i hate using intellij to add parameters with default values for me using the refactor dialog thingo... on the one hand i'm lazy and its nice it can fill the falses in everywhere, but it always mangles everything so idk if i really come out ahead...